src/main/scala/com/lightbend/rp/sbtreactiveapp/magic/Lagom.scala

```diff
@@ -88,19 +88,6 @@ object Lagom {
     }
   }
 
-  def version: Option[String] = {
```

Comment:
Looks like this is public (binary) API. Let's just deprecate it instead of deleting it.

Reply:
Oh… why is any of this class public API? 😕

Reply:
Should I bump version to 1.4.0-SNAPSHOT if I'm deprecating APIs?

Reply:
Sure.